### PR TITLE
Add pitch bend methods to FluidSynth chugin

### PR DESCRIPTION
**Pull request by forrcaho**
A few more hooks into the libfluidsynth API; this time I've done pitch bend.

### CODE REVIEW

**Comment by forrcaho** [comment]
Yeah, I wrote a few more hooks because I'm in it for the Hacktoberfest shirt. Hope this looks OK.

File: FluidSynth/FluidSynth.cpp [hunk 1]
```diff
@@ -263,6 +281,23 @@ CK_DLL_QUERY( fluidsynth )
...
+    QUERY->add_mfun(QUERY, fluidsynth_setPitchBend, "void", "setPitchBend");
+    QUERY->add_arg(QUERY, "int", "pitchbend");
+
+    QUERY->add_mfun(QUERY, fluidsynth_setPitchBendChan, "void", "setPitchBend");
+    QUERY->add_arg(QUERY, "int", "pitchbend");
+    QUERY->add_arg(QUERY, "int", "chan");
+
+    QUERY->add_mfun(QUERY, fluidsynth_setPitchBend, "void", "resetPitchBend");
+
+    QUERY->add_mfun(QUERY, fluidsynth_setPitchBendChan, "void", "resetPitchBend");
```
Comment (lathertonj) [inline FluidSynth/FluidSynth.cpp]: Should this and the one above it be using 
`fluidsynth_resetPitchBend[Chan]`
?

Comment (forrcaho) [reply FluidSynth/FluidSynth.cpp]: Good catch; fix submitted.

**Comment by lathertonj** [comment]
Looks good to me!

**Pull request merged**